src/applications/my-education-benefits/config/form.js

```diff
@@ -154,7 +154,17 @@ function phoneSchema() {
 const formConfig = {
   rootUrl: manifest.rootUrl,
   urlPrefix: '/',
-  // submitUrl: '/v0/api',
+  preSubmitInfo: {
+    ...preSubmitInfo,
+    notice: (
```

Comment:
Is this notice needed? It looks like there's an existing, identical notice in the [platform presubmit component](https://github.com/department-of-veterans-affairs/vets-website/blob/b23f6e1a74a5e4bea58552e2dda0a49d847ad262/src/platform/forms/preSubmitInfo.js#L5).

Reply:
Hey thanks for catching that but I noticed that the [presubmit component](https://github.com/department-of-veterans-affairs/vets-website/blob/b23f6e1a74a5e4bea58552e2dda0a49d847ad262/src/platform/forms/preSubmitInfo.js#L5) styling doesn't fix the VA standard, or in my app.

By default it's pretty compact, a margin top of 1em is what my mock up shows.  

<img width="1522" alt="Screen Shot 2021-09-01 at 10 13 42 AM" src="https://user-images.githubusercontent.com/17830527/131687136-387d22b2-231d-466d-8248-5426cad97693.png">

Reply:
@micahchiang on the left is my component with spacing between NOTE and checkbox, and

right image is the component natural styling.

Reply:
I agree yours looks nicer. 👍  This seems like a change that should be made to the common presubmit component, though.

Reply:
Hey guys, I believe this styling issue was a side effect of styling overrides we made in the My Education Benefits App. I implemented the confirmation checkbox in this PR without the need to customize the preSubmitInfo notice; I just updated our CSS override.

https://github.com/department-of-veterans-affairs/vets-website/pull/18437